README.md

```diff
@@ -82,53 +82,25 @@ compile 'com.okta.spring:okta-spring-boot-starter:${okta.springboot.version}'
 
 where ${okta.springboot.version} is the latest published version in [Maven Central](https://search.maven.org/search?q=g:com.okta.spring%20a:okta-spring-boot-starter).
 
-## Supporting client side applications - OAuth Implicit flow
+## Building API Applications - Resource Server
```

Comment:
We probably want to restructure this Readme a little more after this gets merged in.

The Authorization Code flow is the happy path, that _should_ be the first example, then a Resource Server example, and then if at all the custom hosted example notes should just be a quick link to the examples repo.